lib/stytch/b2b_sessions.rb

```diff
@@ -700,9 +700,9 @@ def authenticate_jwt_local(session_jwt, max_token_age_seconds: nil, authorizatio
       end
 
       # Do the auth check - intentionally don't rescue errors from here
-      if authorization_check && session['roles']
+      if authorization_check && session['scopes']
         @policy_cache.perform_authorization_check(
-          subject_roles: session['roles'],
+          subject_roles: session['scopes'],
```

Comment:
This doesn't look right to me - we still call them `roles` in B2B Member Sessions